packages/valory/skills/abstract_round_abci/behaviours.py

```diff
@@ -168,7 +166,8 @@ def __init__(self, **kwargs: Any) -> None:
         ] = self._get_round_to_behaviour_mapping(self.behaviours)
 
         self.current_behaviour: Optional[BaseBehaviour] = None
-        self.__termination_behaviour: Optional[BaseBehaviour] = None
+        self.termination_behaviour: Optional[BaseBehaviour] = None
+        self._is_termination_set = self.termination_behaviour_cls is not None
```

Comment:
Rather than `termination` I suggest something with `background` or `exit`. I would reserve `termination` for the actual behaviour we are implementing below. Then it is less confusing.